pyproject.toml

```diff
@@ -72,6 +72,7 @@ dependencies = [
   "urllib3>=2.2.3",
   "xmltodict>=0.14.2",
   "openshift-python-wrapper~=4.18.0",
+  "marshmallow~=3.26.1",
```

Comment:
Please add in the PR description why this is needed.
The PR title should be more specific and not general (Update uv packages)
The PR should only add the wanted package without update all other ones.

Reply:
technically, it is a `uv update`. Akriti didn't add `marshmallow` package - it was done automatically, so not sure what else can be added to the title. 
I will mark current comment as resolved, but we will keep in mind for future how to make titles more specific

Reply:
`marshmallow` is pulled as a dependency of `pytest-jira` package and latest 4.0.0 breaks pytest. that's why need to pin it to ver 3.x.x